src/components/Task/Task.css

```diff
@@ -0,0 +1,16 @@
+.task {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding-top: 32px;
+    margin-left: 32px;
+    background-color: #21212B;
+    border-radius: 24px;
+    margin-top: 24px;
```

Comment:
same as the `margin-left`. I wouldn't set margins in a small reusable component like this one